workflow-templates/test-package.yaml

```diff
@@ -0,0 +1,57 @@
+name: Test
```

Comment:
I'd call this `Dart CI` or something; it does a lot more than testing

Reply:
Done.